.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   tests:
     working_directory: ~/circle
     docker:
-      - image: circleci/buildpack-deps:14.04
+      - image: cimg/ruby:2.7.5
```

Comment:
I'm sure this is correct, but I'm going to double-check as all the other replacements have been pretty like-for-like... this seems different to all the others?

Reply:
yes sorry i will add a bit more to this PR message

`buildpack-deps` has been deprecated entirely so there is no convenience image equivalent to move to. that step is only used in running the tests so a ruby 2.7.5 container will do the job as this is just a very small sinatra app. have updated the PR message too.

Reply:
Cool.